keyboards/converter/thinkpad_t6x/rpi_pico/info.json

```
@@ -0,0 +1,134 @@
+{
+    "manufacturer": "strobo5",
+    "keyboard_name": "converter/thinkpad_t6x/rpi_pico",
+    "maintainer": "strobo5",
+    "board": "GENERIC_RP_RP2040",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": false,
+        "mousekey": false,
+        "nkro": false
```

If possible, it's preferable for these to be enabled.
```suggestion
        "extrakey": true,
        "mousekey": true,
```

---

This is interesting, I had ignored these features. Now that I checked them it seems that `extrakey` [should make the volume up/down/mute work](https://docs.qmk.fm/#/faq_misc?id=special-extra-key-doesnt-work-system-audio-control-keys) (any maybe `KC_WBAK` and `KC_WFWD`?). I had forgotten about the fact that they don't.

Now they do! Thanks :)

I also tested the mouse key feature, it works.